src/server/utils/devCdnFactory.js

```diff
@@ -109,7 +110,7 @@ export const oneAppDevCdnFactory = ({
     console.error('do not include one-app-dev-cdn in production');
   }
 
-  const oneAppDevCdn = Fastify();
+  const oneAppDevCdn = Fastify({ logger, disableRequestLogging: true });
```

Comment:
Thoughts on making this configurable for local development? There are times where i'd like to see the requests being made to the dev cdn

Reply:
The dev cdn logs when it receives a request anyway? do we want the double tap?